### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by danielchalmers**
### Description of Change ###

a few tiny changes to the README :)

- XML tab size `4` -> `2`
- `to your nuget` -> `to your NuGet`
- `Nuget.exe` -> `nuget.exe`
- `OSX` -> `macOS`
- update `UWPMapsAuthKey` link
- fix inconsistent markdown headers
- clean up whitespace and newlines

### API Changes ###

 
 None

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

Not applicable

### PR Checklist ###

- [x] Has automated tests 
- [x] Rebased on top of the target branch at time of PR
- [x] Changes adhere to coding standard

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,23 +69,25 @@ Due to the way that Android works, the maps API key cannot be injected at runtim
...
 ##### Build from the Command line #####
-Make sure you have Nuget.exe 4.0 or above and the latest dotnet core sdk (2.0.3). On OSX you should specify the platform in the msbuild command (`/p:Platform=iPhoneSimulator`)
-  
 
-     nuget restore Xamarin.Forms.sln
-     msbuild Xamarin.Forms.sln
- 
+Make sure you have nuget.exe 4.0 or above and the latest dotnet core sdk (2.0.3). On macOS you should specify the platform in the msbuild command (`/p:Platform=iPhoneSimulator`)
+
+    nuget restore Xamarin.Forms.sln
```
Comment (VladislavAntonyuk) [inline README.md]: Should be in {code}

File: README.md [hunk 2]
```diff
@@ -60,7 +60,7 @@ By default, the `Xamarin.Forms.Controls` project does not have a configuration f
 
 You will have to obtain your own API keys for each of these services, inserted directly after the identifier (e.g. `UWPMapsAuthKey:abcdefghijklmnopqrstuvwxyz`). You can find out how to obtain each of these as follows:
 
-- `UWPMapsAuthKey` at https://www.microsoft.com/maps/create-a-bing-maps-key.aspx
+- `UWPMapsAuthKey` at https://microsoft.com/maps/create-a-bing-maps-key
```
Comment (davidortinau) [inline README.md]: this updated URL doesn't take me to a page where I can create a key. The existing URL does. What am I missing here?

File: README.md [hunk 3]
```diff
@@ -7,42 +7,41 @@ Xamarin.Forms provides a way to quickly build native apps for iOS, Android, Wind
 
 Read more about the platform at https://www.xamarin.com/forms.
 
-## Build Status 
+## Build Status ##
 
-![Visual Studio Team Services ](https://img.shields.io/vso/build/devdiv/0bdbc590-a062-4c3f-b0f6-9383f67865ee/7981.svg?style=flat&label=VSTS%20Master "VSTS")
- 
+![Visual Studio Team Services](https://img.shields.io/vso/build/devdiv/0bdbc590-a062-4c3f-b0f6-9383f67865ee/7981.svg?style=flat&label=VSTS%20Master "VSTS")
```
Comment (davidortinau) [inline README.md]: Probably should replace this with Azure DevOps. @rmarinho would this work or does the label on the SVG need to change on the build system as well?
```suggestion
![Azure DevOps](https://img.shields.io/vso/build/devdiv/0bdbc590-a062-4c3f-b0f6-9383f67865ee/7981.svg?style=flat&label=Azure%DevOps%20Master "Azure DevOps")
```

Comment (rmarinho) [reply README.md]: https://img.shields.io/vso/build/devdiv/0bdbc590-a062-4c3f-b0f6-9383f67865ee/7981.svg?style=flat&label=AzureDevOps%20Master

Comment (rmarinho) [reply README.md]: the url is wrong o the label part.

**Pull request merged**